### DISCUSSION TITLE
Where are all the STL file for Create3 camera mounts,  upper deck, etc ?

### DISCUSSION THREAD

**Discussion by jerrynsr** [iRobotEducation/create3_docs#50] [Questions] [answered]
Before I ordered the Create3 I can swear I found STL files for a  camera mount, Oak D lite mount,  Slamtec A1 mount.. and more on 
irobot's edu website.  I can't find it now.  Does anyone here have the link ?

**Comment by shamlian** [comment, marked as answer]
We're currently refactoring some of our tutorials, but the STLs that are available can be found directly in the [source](https://github.com/iRobotEducation/create3_docs/tree/main/docs/hw/data/brackets). Expect to see more content in our [docs](https://iroboteducation.github.io/create3_docs/) site referencing these STLs, soon.

**Reply by shamlian** [reply]
Pre-release STLs for OAK-D and OAK-D-Lite are now in that same directory. I was able to test the OAK-D-Lite STL but I don't have an OAK-D for a fit check.
